spec/features/trainees/viewing_an_existing_trainee.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature "View trainees", type: :feature do
+
```

Comment:
Whitespace line. Not sure why the linter isn't picking this up. I'm sure the other apps flag these whitespace lines.